code/modules/paperwork/records/info/he.dm

```diff
@@ -136,14 +133,12 @@
 	Qliphoth Counter : 4    <br>
 	Work Damage Type : Black	<br>
 	Work Damage : High	<br>
-	- When two employees died, the Qliphoth Counter lowered.    <br>
+	- After 2 employees died within the facility, the Qliphoth Counter lowered.    <br>
 	- When Repression work was completed, the Qliphoth Counter increased.    <br>
-	- When Release work was completed, Blue Smocked Shepard breached,    <br>
+	- When Release work was completed, Blue Smocked Shepard escaped its Containment Unit,    <br>
 	- When any other work was completed, the Qliphoth Counter lowered. <br>
-	- Blue Smocked Shepard mercilessly attacked everyone in the area.<br>
-	- If Reddened Buddy is in the facility, his work chance increase proportionally to the amount of pain Reddened Buddy is experiencing <br>
-	- If Reddened Buddy's pain was too high however, the Qliphoth Counter lowered on work completion <br>
-	- Blue Smocked Shepard mercilessly attacked everyone in the area. <br>
```

Comment:
This is here to mention that Blue Shep is a neutral mob and that it's not strictly a bad thing to release it.

Reply:
I only request this because you can release Blue Shep on purpose.